runners/google-cloud-dataflow-java/src/test/java/org/apache/beam/runners/dataflow/DataflowPipelineTranslatorTest.java

```diff
@@ -1036,6 +1036,32 @@ public void testToIterableTranslationWithIsmSideInput() throws Exception {
     assertEquals("CollectionToSingleton", collectionToSingletonStep.getKind());
   }
 
+  @Test
+  public void testToIterableTranslationFnApiSideInput() throws Exception {
+    // A "change detector" test that makes sure the translation
```

Comment:
Unless this test is to ensure update compatibility I would prefer to avoid this test, as I don't think it actually provides any value. If that is the case, please comment appropriately.

Reply:
Dropped